spec/mailers/claim_feedback_mailer_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# rubocop:disable RSpec/MultipleMemoizedHelpers
+RSpec.describe ClaimFeedbackMailer, type: :mailer do
```

Comment:
I would love to see this functionality covered by an integration spec as well as these unit specs, to 'prove' that all the moving parts join up as expected - something that says effectively "Given there is a claim, when a caseworker clicks the button to approve it, then an email is triggered to the provider". I know the 'what's our test strategy' conversation is still in flight, though, so I don't want to rock the boat while we don't have an agreed convention.

Reply:
I will look at adding this in before proceeding with more reviews. Its good we try and cover this now rather than miss it